.travis.yml

```diff
@@ -48,23 +43,17 @@ before_install:
         pecl install pcov
       fi
 
-  - |
-      if [[ $TRAVIS_PHP_VERSION != 'nightly' ]]; then
```

Comment:
Can we just remove the nightly include and leave this so future merges are clean?

Reply:
Done